images/fcos-bastion-image/root/usr/share/bastion_services/ignitions/wipe_disks.bu

```diff
@@ -0,0 +1,27 @@
+variant: fcos
+version: 1.5.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys_local:
+        - authorized_keys
+systemd:
+  units:
+    - name: wipe-disks.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Wipe the disks
+        OnFailure=emergency.target
+        OnFailureJobMode=replace-irreversibly
+        After=network-online.target
+        Wants=network-online.target
+        [Service]
+        Type=oneshot
+        ExecStart=bash -c 'for i in ; do dd if=/dev/zero of=/dev/ bs=1M count=16; done'
```

Comment:
something is missing here. Perhaps, `$(lsblk -I8,259 -nd --output name)`  after `in`. Double check the whole file

Reply:
Fixed. good catch!